### PR TITLE
Create Well-Known-Specification.md

### PR DESCRIPTION
**Pull request by sjledoux**
In order to register the `/.well-known/related-website-set.json` URI, `Well-Known-Specification.md` should be created so that we have a dedicated reference page that describes the purpose and content of the `/.well-known/related-website-set.json` file.

### CODE REVIEW

**Review by helenyc — APPROVED** [review]

File: Well-Known-Specification.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# The "related-website-set.json" Well-Known Resource Identifier
+
+This page describes the use and content of URIs having the path 
+`/.well-known/related-website-set.json`.
+## Use
+
+A page may be hosted at the URI having the path of 
+`/.well-known/related-website-set.json` in order to demonstrate
+administrative access to the site when submitted as an entry in a Related 
+Website Site (RWS).
```
Comment (cfredric) [inline Well-Known-Specification.md:10]: Please use active voice to make this clearer:

```suggestion
A site may demonstrate intentional opt-in to participating in a Related Website Set (RWS) by hosting a specific JSON file at the URI with a path of
`/.well-known/related-website-set.json`.
```

File: Well-Known-Specification.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# The "related-website-set.json" Well-Known Resource Identifier
+
+This page describes the use and content of URIs having the path
```
Comment (cfredric) [inline Well-Known-Specification.md:3]:
```suggestion
This page describes the use and content of URIs with the path 
```

Nit: "having" feels like odd word choice to me.

File: Well-Known-Specification.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+`/.well-known/related-website-set.json`.
+## Use
+
+A page may be hosted at the URI having the path of 
+`/.well-known/related-website-set.json` in order to demonstrate
+administrative access to the site when submitted as an entry in a Related 
+Website Site (RWS).  
+## Content
+
+The `/.well-known/related-website-set.json` file for the set primary of an RWS
```
Comment (cfredric) [inline Well-Known-Specification.md:13]: Nit: trailing whitespace

```suggestion
The `/.well-known/related-website-set.json` file for the set primary of an RWS
```

File: Well-Known-Specification.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  "dependentRequired": {
+    "associatedSites": ["An explanation of how you clearly present the 
+    affiliation across domains to users and why users would expect your 
+    domains to be affiliated"],
+    "serviceSites": ["An explanation of how each domain in this subset 
+    supports functionality or security needs."]
+  }
+}
+```
+The `/.well-known/related-website-set.json` file for set members of an RWS
```
Comment (cfredric) [inline Well-Known-Specification.md:58]:
```suggestion
The `/.well-known/related-website-set.json` file for set members of an RWS
```

File: Well-Known-Specification.md (hunk 1 above)
Comment (cfredric) [inline Well-Known-Specification.md:10]:
```suggestion
Website Site (RWS).
```

File: Well-Known-Specification.md (hunk 4 above)
Comment (cfredric) [inline Well-Known-Specification.md:58]:
```suggestion
The `/.well-known/related-website-set.json` file for non-primary members of an RWS 
```

"set members of an RWS" feels redundant, and I think we also need to specify that we're not talking about the primary sites.

File: Well-Known-Specification.md (hunk 1 above)
Comment (sjledoux) [reply Well-Known-Specification.md:10]: Done

**Review by cfredric — APPROVED** [review]
LGTM, thanks!

**Review by mostafakn — COMMENTED** [review]
Well-Known-Specification.md

**Pull request merged**